src/manifold/src/manifold.cpp

```diff
@@ -71,6 +71,12 @@ Manifold::Manifold(std::shared_ptr<CsgNode> pNode) : pNode_(pNode) {}
 Manifold::Manifold(std::shared_ptr<Impl> pImpl_)
     : pNode_(std::make_shared<CsgLeafNode>(pImpl_)) {}
 
+Manifold Manifold::Invalid() {
```

Comment:
I think we have a `MarkFailure` method somewhere; could we use that instead?

Reply:
This one? https://github.com/elalish/manifold/blob/2c597d1eefea3febad4efe6683f3df0f44b7a9df/src/manifold/src/impl.h#L77 

It returns void, so still needs an impl created before applying it. Do you mean use it in the Invalid helper instead of setting the status directly? (I made the private helper just so the same lines wouldn't be repeated each time)

Reply:
Ah, fair, that's not as useful as I was hoping. This is fine.